data/json/monsters.json

```diff
@@ -1342,6 +1342,35 @@
     "anger_triggers":["HURT", "FRIEND_DIED", "PLAYER_CLOSE"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_wasp_small",
+    "//": "id mon_wasp is already used, so changing would break tilests and other things",
+    "name": "wasp",
+    "species":"INSECT",
+    "default_faction":"wasp",
```

Comment:
Faction "wasp" will probably make other creatures (zombies!) target the tiny wasp. Faction "vermin" should prevent it.